local/compose/images.go

```diff
@@ -66,7 +68,9 @@ func (s *composeService) Images(ctx context.Context, projectName string, options
 			if err != nil {
 				return err
 			}
+			l.Lock()
```

Comment:
Why not just use https://golang.org/pkg/sync/#Map ? ☺️

Reply:
Yes, actually a good suggestion... I was about to do this, and then Map is totally untyped, so I prefer the 3 lines of explicit locking here, and not have to cast things further down (and the doc says "Most code should use a plain Go map instead, with separate locking" 🙄  )